### DISCUSSION TITLE
Build failed with Android Studio on fresh app

### DISCUSSION THREAD

**Discussion by buhrmi** [ionic-team/capacitor#5123] [Q&A]
Hi all, 

after opening a completely new project (generated with `npx cap add android` and `npx cap sync`) I get this error when clicking the run button `No matching configuration of project :capacitor-android was found`:
![image](https://user-images.githubusercontent.com/51177/136811566-ed945779-789f-42cd-9463-3beac629a53f.png)

I've deleted and recreated the project a couple of times, but the error still appears. Is there anything I might be missing?

**Comment by buhrmi** [comment]
In case anyone is interested in looking at this, here is the zipped project. I zipped it up directly after `npx cap add android`. I can't figure out what's wrong...
[android.zip](https://github.com/ionic-team/capacitor/files/7323807/android.zip)

**Reply by cguerrero73** [reply]
Hi buhrmi, you have some solution to this error, i have exactly the same error. Thanks

**Reply by cguerrero73** [reply]
I found my problem, the android folder was generated in wsl and copied to windows folder to build the apk with android studio, is necesary the node_modules folder
